src/lib/DpApi.js

```diff
@@ -75,10 +75,16 @@ const doRequest = (async ({ method = 'GET', url, data = {}, headers, params }) =
       ? await response.text()
       : null
 
-    return {
+    const returnData = await checkResponse({
       data: content,
-      status: response.status,
       ok: response.ok,
+      status: response.status
+    }, options.messages || null)
```

Comment:
Where does this "options" come from? Seems like it got removed in the signature of `doRequest` in a recent refactoring, and this was not updated accordingly.